src/helpers/userHelper.js

```diff
@@ -171,6 +172,14 @@ const userHelper = function () {
           ? `<a href="${mediaUrl}">${mediaUrl}</a>`
           : 'Not provided!';
 
+        const googleDocLinkValue = adminLinks 
```

Comment:
This line: const googleDocLinkValue = adminLinks ? ... : .....
SHoul be: const googleDocLinkValue = adminLinks.length > 0 ? ... : .... 
Because the  adminLinks variable is an array. Even when it is empty, it still evaluates to truthy. This means that the ternary operation will always return true.

Reply:
Hello, @abdel-lall! Thank you for your review! I agree with your suggestions and have maded the necessary changes to the implementation, Requesting your approval upon testing the updated PR. Thank you!